Firestore/Swift/Tests/Codable/CodableDocumentTests.swift

```diff
@@ -20,12 +20,12 @@ import Foundation
 import XCTest
 
 class CodableDocumentTests: XCTestCase {
-  func roundTrip<X>(input: X, expected: [String: Any], doTest: Bool = true) -> X where X: Codable {
+  func roundTrip<X>(input: X, expected: [String: Any]? = nil) -> X where X: Codable {
     var encoded = [String: Any]()
     do {
       encoded = try Firestore.Encoder().encode(input)
-      if doTest {
-        XCTAssertEqual(encoded as NSDictionary, expected as NSDictionary)
+      if expected != nil {
```

Comment:
We can get rid of the force unwrap:

```
if let expected = expected {
  XCTAssertEqual(encoded as NSDictionary, expected as NSDictionary)
}
```

If it doesn't let you redeclare the variable (I can never remember where it's disallowed...) feel free to call it whatever you think makes the most sense.